src/datadog/azext_datadog/generated/_params.py

```diff
@@ -19,43 +19,24 @@
 )
 from azure.cli.core.commands.validators import get_default_location_from_resource_group
 from azext_datadog.action import (
+    AddMarketplaceagreementsProperties,
     AddDatadogOrganizationProperties,
     AddUserInfo,
-    AddMetricRulesFilteringTags,
+    AddFilteringTags,
     AddLogRulesFilteringTags,
-    AddProperties
+    AddSinglesignonconfigurationsProperties
 )
 
 
 def load_arguments(self, _):
 
-    with self.argument_context('datadog api-key list') as c:
-        c.argument('resource_group_name', resource_group_name_type)
-        c.argument('monitor_name', type=str, help='Monitor resource name')
-
-    with self.argument_context('datadog api-key get-default-key') as c:
-        c.argument('resource_group_name', resource_group_name_type)
-        c.argument('monitor_name', type=str, help='Monitor resource name', id_part='name')
-
-    with self.argument_context('datadog api-key set-default-key') as c:
-        c.argument('resource_group_name', resource_group_name_type)
-        c.argument('monitor_name', type=str, help='Monitor resource name', id_part='name')
-        c.argument('created_by', type=str, help='The user that created the API key.')
-        c.argument('name', type=str, help='The name of the API key.')
-        c.argument('key', type=str, help='The value of the API key.')
-        c.argument('created', type=str, help='The time of creation of the API key.')
-
-    with self.argument_context('datadog host list') as c:
-        c.argument('resource_group_name', resource_group_name_type)
-        c.argument('monitor_name', type=str, help='Monitor resource name')
+    with self.argument_context('datadog marketplace-agreement create') as c:
+        c.argument('properties', action=AddMarketplaceagreementsProperties, nargs='+', help='Represents the properties '
+                   'of the resource.')
```

Comment:
Sorry, I forgot to check in the `readme.az.md` in the main repo last time. I just submitted a PR https://github.com/Azure/azure-rest-api-specs/pull/13696.   The `marketplace-agreement` group is renamed to `terms`. Can you apply the change locally and regenerate the code.